docs/adminguide/hosting/parsec.env

```diff
@@ -0,0 +1,34 @@
+# Host to listen to.
+PARSEC_HOST=0.0.0.0
+
+# The SSL key file.
+PARSEC_SSL_KEYFILE=/run/secrets/parsec-pem-key
+# The SSL certificate file.
+PARSEC_SSL_CERTFILE=/run/secrets/parsec-pem-crt
+# Enforce HTTPS by redirecting HTTP request.
+PARSEC_FORWARD_PROTO_ENFORCE_HTTPS=X-Forwarded-Proto:https
+
+# The granularity of Error log outputs.
+PARSEC_LOG_LEVEL=WARNING
+# The log formatting to use (`CONSOLE` or `JSON`).
+PARSEC_LOG_FORMAT=CONSOLE
+# The log file to write to (default to `stderr`).
+# PARSEC_LOG_FILE
+
+# The URL to reach this server (typically used in invitation emails).
```

Comment:
Why talk about invitation emails? Isn't used for everything requiring accessing Parsec server? 
```suggestion
# The URL to reach Parsec server
```